unstructured/ingest/README.md

```diff
@@ -71,6 +75,13 @@ In checklist form, the above steps are summarized as:
   - [ ] The added dependencies should be imported at runtime when the new connector is invoked, rather than as top-level imports.
   - [ ] Add the decorator `unstructured.utils.requires_dependencies` on top of each class instance or function that uses those connector-specific dependencies e.g. for `GitHubConnector` should look like `@requires_dependencies(dependencies=["github"], extras="github")`
   - [ ] Run `make tidy` and `make check` to ensure linting checks pass.
+- [ ] Update ingest documentation [here](https://github.com/Unstructured-IO/unstructured/tree/eb8ce8913729826b62fd4e1224f70d67c5289b9d/docs/source)
+- [ ] If there are secret variables obtained for the connector tests, make sure to:
+  - [ ] contact Unstructured team for them to add the secrets into Github
+  - [ ] include the secret variables in [`ci.yml`](https://github.com/Unstructured-IO/unstructured/blob/eb8ce8913729826b62fd4e1224f70d67c5289b9d/.github/workflows/ci.yml) and [`ingest-test-fixtures-update-pr.yml`](https://github.com/Unstructured-IO/unstructured/blob/eb8ce8913729826b62fd4e1224f70d67c5289b9d/.github/workflows/ingest-test-fixtures-update-pr.yml)
+  - [ ] add a make install line in the workflow configurations to be able to provide the workflow machine with the required dependencies on the connector while testing
+- [ ] Make sure that the tests for the connector are running and not getting skipped.
+- [ ] Whenever necessary, use the [ingest update test fixtures](https://github.com/Unstructured-IO/unstructured/actions/workflows/ingest-test-fixtures-update-pr.yml) workflow to update the test fixtures.
```

Comment:
Unfortunately this isn't available to contributors developing on a fork, because this relies on secrets that are only available in the upstream repository.

Reply:
Should I group these items under a line named "additional steps for team members", or should we remove it?

Reply:
I tried to address this with https://github.com/Unstructured-IO/unstructured/pull/1456/commits/6ec46b68d5463ba734ce9c7be383945a223ea9ae, please let me know if a different approach would be better